web/src/main/webapp/WEB-INF/jsp/team/detail/results.jsp

```diff
@@ -4,7 +4,7 @@
 <%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
 <%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
 <%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
-<%--@elvariable id="team" type="cz.muni.fi.pa165.team.Team"--%>
```

Comment:
když tady dáš takovouto annotaci pro ty proměnné, tak ono to pak v té šabloně hezky napovídá, takže bych to nemazal a naopak tam přidal další pro ty nové proměnné co přibyly